README.md

```diff
@@ -10,3 +10,4 @@ Where the following can be found:
 - Getting started guides
 - Flotta Operations
 - Dev Guides
+1
```

Comment:
I guess this shouldn't be here :-) 
Removed by https://github.com/project-flotta/flotta-operator/pull/276